package.json

```diff
@@ -95,7 +99,7 @@
     "@commitlint/prompt": "^16.2.3",
     "@microsoft/api-extractor": "^7.33.5",
     "@types/glob": "^7.1.1",
-    "@types/jest": "^26.0.20",
+    "@types/jest": "^27.0.0",
```

Comment:
the upgrade to Jest 27 fixes several issues with worker pools:
https://github.com/facebook/jest/issues/11143